docs/_layouts/default.html

```diff
@@ -32,6 +32,7 @@
     {{ content }}
     {% include footer.html %}
   </div>
-  <script src="{{ site.baseurl }}/dist/js/design-system.js" charset="utf-8"></script>
+  <script src="{{ site.baseurl }}/dist/js/main.js" charset="utf-8"></script>
+  {% include search-scripts.html %}
```

Comment:
Do we need the index present on every page? Doesn't `search-scripts.html` only have to exist on `search-results.html`? The long build time is because we're generating the search index 60+ times (once for every page).

Reply:
Hmm, yeah I think I was thinking we might have an autocomplete or something ajax-y on the search at some point and plopped it in the default.html for that reason, but you're right, I'll move it.

Reply:
Moved in https://github.com/cfpb/design-system/pull/458/commits/005a9f338328b4d8546a22ba4de49d1928392789

Reply:
Cool and if we do an autocomplete in the future we can export the index as a json file and asynchronously fetch it.